src/util/misc.ts

```diff
@@ -283,7 +283,7 @@ export function getStrippedTitleFromHtml(html: string) {
   const doc = domino.createDocument(html)
```

Comment:
I'm choqued by this line which is super CPU expensive. There is no way we coukd do that quicker?! I know this is not the topic of this PR, but stil...